aws-rds-customdbengineversion/aws-rds-customdbengineversion.json

```diff
@@ -0,0 +1,174 @@
+    "InitechDateFormat": {
+      "$comment": "Use the `definitions` block to provide shared resource property schemas",
+      "type": "string",
+      "format": "date-time"
+    },
+    "Memo": {
+      "type": "object",
+      "properties": {
+        "Heading": {
+          "type": "string"
+        },
+        "Body": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "Tag": {
+      "description": "A key-value pair to associate with a resource.",
+      "type": "object",
+      "properties": {
+        "Key": {
+          "type": "string",
+          "description": "The key name of the tag. You can specify a value that is 1 to 128 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -.",
+          "minLength": 1,
+          "maxLength": 128
+        },
+        "Value": {
+          "type": "string",
+          "description": "The value for the tag. You can specify a value that is 0 to 256 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -.",
+          "minLength": 0,
+          "maxLength": 256
+        }
+      },
+      "required": [
+        "Key",
+        "Value"
+      ],
+      "additionalProperties": false
+    }
+  },
+  "properties": {
+    "DatabaseInstallationFilesS3BucketName": {
+      "type": "string",
+      "description": "The name of an Amazon S3 bucket that contains database installation files for your CEV. For example, a valid bucket name is `my-custom-installation-files`.",
+      "minLength": 3,
+      "maxLength": 63
+    },
+    "DatabaseInstallationFilesS3Prefix": {
+      "type": "string",
+      "description": "The Amazon S3 directory that contains the database installation files for your CEV. For example, a valid bucket name is `123456789012/cev1`. If this setting isn't specified, no prefix is assumed.",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "Description": {
+      "type": "string",
+      "description": "An optional description of your CEV.",
+      "minLength": 1,
+      "maxLength": 1000
+    },
+    "Engine": {
+      "type": "string",
+      "description": "The database engine to use for your custom engine version (CEV). The only supported value is `custom-oracle-ee`.",
+      "pattern": "^[A-Za-z0-9-]{1,35}$",
+      "minLength": 1,
+      "maxLength": 35
+    },
+    "EngineVersion": {
+      "type": "string",
+      "description": "The name of your CEV. The name format is 19.customized_string . For example, a valid name is 19.my_cev1. This setting is required for RDS Custom for Oracle, but optional for Amazon RDS. The combination of Engine and EngineVersion is unique per customer per Region.",
+      "pattern": "^(11\\.\\d{1}|12\\.\\d{1}|18|19)(\\.[a-zA-Z0-9_.-]{1,50})$",
```

Comment:
Let's remove this validation as well. The example provides a sufficient instruction on the formatting rules. The rest is between the customer and the RDS API.